### PR TITLE
Fix build (reloaded)

### PR DESCRIPTION
**Pull request by mackwic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
-script:
-  - bash -ex .travis-ci.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+
+script: bash -ex .travis-opam.sh
 
 env:
-  - OCAML_VERSION=4.01.0 OPAM_VERSION=1.2.0
-  - OCAML_VERSION=4.02.1 OPAM_VERSION=1.2.0
+  - OCAML_VERSION=latest
```
Comment (mc10) [inline .travis.yml]: Might want to pin one or two versions (like 4.06) instead of using `latest`.

File: .travis.yml [hunk 2]
```diff
@@ -2,13 +2,17 @@
 # based on http://anil.recoil.org/2013/09/30/travis-and-ocaml.html
 
 language: c
+sudo: required
 
-script:
-  - bash -ex .travis-ci.sh
```
Comment (mc10) [inline .travis.yml]: Since you're removing this line, you can probably delete this file as well.

Comment (mackwic) [reply .travis.yml]: you are right !

**Comment by mackwic** [comment]
thank you for your feedback @mc10 ! I'll let this PR one day or two then I'll merge if everyone is OK

File: .travis.yml (hunk 1 above)
Comment (mackwic) [reply .travis.yml]: doesn't work, no time to look at it. So Lastest it will be until someone changes it

**Pull request merged**